L1Trigger/Phase2L1GMT/test/makeKMTFCoarsseEtaLUT.py

```diff
@@ -0,0 +1,13 @@
+import ROOT
+from math import pi
+f=ROOT.TFile("../data/packedGainLUTs.root")
```

Comment:
I am not sure if, in general, this will work in a standard CMSSW release. An edm `FileInPath` may work?

Reply:
Since this is a standalone python script, we think it is OK as it is.